actioncable/lib/action_cable.rb

```diff
@@ -37,6 +37,10 @@ module ActionCable
     },
     default_mount_path: '/cable'.freeze
   }
+  
+  # This protocol ought to be versioned separately
+  # from the rest of the Gem
+  PROTOCOLS = ["actioncable-v1-json".freeze].freeze
```

Comment:
Do we want this to have the same version number as the gem? Would prevent having two different version numbers for Action Cable...